src/tests/managed_array_tests.cpp

```diff
@@ -410,3 +409,12 @@ CUDA_TEST(ManagedArray, Move)
   array.free();
 }
 #endif // defined(CHAI_ENABLE_CUDA)
+
```

Comment:
@artv3 I think that the intention was to use @davidbeckingsale tests. Can you revert this file to his commit?

Reply:
Should I remove yours? Or keep them as well?

Reply:
I think David's tests cover the space better...right?